### PR TITLE
Adding flaky test suite.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Add test suite which would run flaky tests. That would allow us to gather more data what is current state of the product. 
There is a PR to openshiftio-cico-jobs https://github.com/openshiftio/openshiftio-cico-jobs/pull/989. Once this is merged, the job should be created and test would run every 6 hours against production environment with cluster 2.

### CODE REVIEW

**Comment by ibuziuk** [comment]
flacky = che 7 tests atm right ?

**Comment by Katka92** [comment]
@ibuzliuk Yes, Che 7 is now flaky and I don't want to have it in a PR check or periodic suites. But this flaky suite can be used for every new tests to see if it is stable before putting it to PR check or periodic suites.

File: functional-tests/devscripts/run_tests.sh [hunk 1]
```diff
@@ -79,20 +85,21 @@ else
...
-	           -e "CHE_OSIO_AUTH_ENDPOINT=$CHE_OSIO_AUTH_ENDPOINT" \
-	           -e "RHCHE_HOST_URL=$HOST_URL" \
-           quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
-    RESULT=$?
+		docker run --name functional-tests-dep --privileged \
+	    	       -v /var/run/docker.sock:/var/run/docker.sock \
+	        	   -v /root/payload/logs:/root/logs \
+		           -e "RHCHE_SCREENSHOTS_DIR=/root/logs/screenshots" \
+		           -e "RHCHE_ACC_USERNAME=$USERNAME" \
+	    	       -e "RHCHE_ACC_PASSWORD=$PASSWORD" \
```
Comment (garagatyi) [inline functional-tests/devscripts/run_tests.sh]:
```suggestion
	    	           -e "RHCHE_ACC_PASSWORD=$PASSWORD" \
```

File: functional-tests/devscripts/run_tests.sh [hunk 2]
```diff
@@ -79,20 +85,21 @@ else
...
+	        	   -e "RHCHE_ACC_EMAIL=$EMAIL" \
+		           -e "CHE_OSIO_AUTH_ENDPOINT=$CHE_OSIO_AUTH_ENDPOINT" \
+		           -e "RHCHE_HOST_URL=$HOST_URL" \
+	    	       -e "TEST_SUITE=$TEST_SUITE" \
```
Comment (garagatyi) [inline functional-tests/devscripts/run_tests.sh]:
```suggestion
	    	           -e "TEST_SUITE=$TEST_SUITE" \
```

**Pull request merged**